algrtm/src/app/contactUs/page.js

```diff
@@ -1,14 +1,13 @@
 import ContactDescription from "@/components/ContactDescription";
 import ContactFormContainer from "@/components/ContactFormContainer";
-
 import SolutionsSection from "@/components/SolutionsSection";
 import RoundedRectangle from "@/components/shared/RoundedRectangle";
 import React from "react";
 
 const ContactUs = () => {
   return (
-    <section className="flex flex-col items-start mt-[4.94rem] ">
-      <div className="flex items-start gap-[4rem] mx-[12.5rem]">
+    <section className="flex flex-col items-start mt-[2.75rem] md:mt-[4rem] xl:mt-[4.94rem]">
+      <div className="md:flex flex-row flex-column items-center gap-[2.75rem] md:gap-[4rem] mx-[1rem] md:mx-[2.75rem] xl:mx-[12.5rem] px-4">
```

Comment:
There is a problem with this view on medium-sized devices, as the form does not stretch to its full width:

![image](https://github.com/Nospoko/algrtm/assets/26410548/6abb2b43-c6e1-44c5-8ce4-97e59b94e9d9)